src/tests/ftest/util/ddb_utils.py

```diff
@@ -60,13 +61,11 @@ def run(self):
         """Run the command.
 
         Returns:
-            list: A list of dictionaries with each entry containing output, exit status,
-                and interrupted status common to each group of hosts.
+            CommandResult: groups of command results from the same hosts with the same return status
```

Comment:
We also need to update the docstrings for a few DdbCommand methods:

- list_component
- value_dump
- value_load
- remove_component
- ilog_dump
- ilog_commit
- ilog_clear
- superblock_dump
- dtx_dump
- dtx_cmt_clear